chrome-extension/src/pages/app/app.tsx

```diff
@@ -2,44 +2,55 @@ import { useEffect, useState } from 'react';
 import './App.css'
 import NetworkTable from './components/NetworkTable.tsx'
 import WaterfallChart from './components/WaterfallGraph.tsx';
-import checkTraceId from './functions/checkTraceId.ts'
-import { OtelData } from '../../types/types.ts';
+import {IRelevant} from '../../types/types.ts';
+import {isRelevant} from './functions/isRelevant.ts';
+import ClearState from './components/ClearState.tsx';
+import Reload from './components/Reload.tsx';
 
 function App() {
-  const [networkRequests, setNetworkRequests] = useState<OtelData[]>([]);
+  const [relevant, setRelevant] = useState<IRelevant>(new Map()); 
 
+
+  // testing uses only: to cross-reference incoming client data with otel data
   useEffect(() => {
     chrome.runtime.onMessage.addListener((message) => {
-      console.log('received chrome message', message)
+      // console.log(`chromeApiRequestData: ${message.type}, startTime ${message.startTime}`, message);
```

Comment:
I noticed a lot of console logs that are still in the code, but commented out. Would these be fine to take out?